Documentation/planning/arcade-powered-source-build/implementation-plan.md

```diff
@@ -5,32 +5,38 @@ To get each repo building with the new source-build 5.0 plan, [Arcade-Powered So
 > ![](img/implementation-plan-graph.png)
 > [source (img/implementation-plan-graph.dgml)](img/implementation-plan-graph.dgml)
 
-| Tier | Repo | (Stage 1)<br>Build from source - 5.0 | (Stage 2)<br>Patch Removal | (Stage 3)<br>Move build params to eng dir | (Stage 4)<br>Input Intermediate Packages Available | (Stage 5)<br>Official build | (Stage 6)<br>Source-build PR validation | (Stage 7)<br>Prebuilt check / enforcement |
-| --- | --- | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
-| sbrp | Source-build-reference-packages | ✔️ | ✔️ | | | | | |
-| Tools | sourcelink | ✔️ | 4 | | | | | |
-| Tools | arcade | ✔️ | 11 | | | | | |
-| 1 | application-insights | ✔️ | ✔️ | | | | | |
-| 1 | aspnet-xdt | ✔️ | 2 | | | | | |
-| 1 | newtonsoft-json | ✔️ | ✔️ | | | | | |
-| 1 | netcorecli-fsc | ✔️ | ✔️ | | | | | |
-| 1 | newtonsoft-json901 | ✔️ | ✔️ | | | | | |
-| 1 | xliff-tasks | ✔️ | 1 | | | | | |
-| 1 | clicommandlineparser | ✔️ | 1 | | | | | |
-| 1 | roslyn | ✔️ | 4 | | | | | |
-| 2 | linker | ✔️ | 3 | | | | | |
-| 2 | runtime | ✔️ | 10 | | | | | |
-| 2 | msbuild | ✔️ | 4 | | | | | |
-| 2 | NuGet.Client | ✔️ | 8 | | | | | |
-| 2 | templating |  | ✔️ | | | | | |
-| 3 | extensions |  | 1 | | | | | |
-| 3 | aspnetcore-tooling |  | 3 | | | | | |
-| 3 | aspnetcore |  | 11 | | | | | |
-| 3 | websdk |  | 1 | | | | | |
-| 4 | sdk |  | 2 + 3(cli) + 3(toolset) | | | | | |
-| 4 | vstest |  | 6 | | | | | |
-| 4 | fsharp |  | 2 | | | | | |
-| 5 | installer |  | 5 | | | | | |
+| Tier | Repo | Owner | (Stage 1)<br>Build from source - 5.0 | (Stage 2)<br>Patch Removal | (Stage 3)<br>Move build params to eng dir | (Stage 4)<br>Input Intermediate Packages Available | (Stage 5)<br>Official build | (Stage 6)<br>Source-build PR validation | (Stage 7)<br>Prebuilt check / enforcement |
+| --- | --- | --- | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
+| sbrp | Source-build-reference-packages | Dan Seefeldt | ✔️ | ✔️ | | | | | |
+| Tools | sourcelink | Tomas Matousek | ✔️ | 4 | | | | | |
+| Tools | arcade | Mark Wilkie | ✔️ | 11 | | | | | |
+| 1 | application-insights | Reiley Yang | ✔️ | ✔️ | | | | | |
+| 1 | aspnet-xdt | Vijay Ramakrishnan | ✔️ | 2 | | | | | |
+| 1 | newtonsoft-json | Chris Rummel | ✔️ | ✔️ | | | | | |
+| 1 | netcorecli-fsc | Chris Rummel | ✔️ | ✔️ | | | | | |
+| 1 | newtonsoft-json901 | Chris Rummel | ✔️ | ✔️ | | | | | |
```

Comment:
Add GitHub alias, or make the name a link to the GitHub account to save space?

Reply:
I thought about using aliases, but thought I liked full names better.

Reply:
Did you consider linking to the account so the alias is at least accessible?

Reply:
https://github.com/dotnet/source-build/pull/1623